src/tests/sqlsmith/tests/freeze/ddl.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TABLE alltypes1 (c1 BOOLEAN, c2 SMALLINT, c3 INT, c4 BIGINT, c5 REAL, c6 DOUBLE, c7 NUMERIC, c8 DATE, c9 CHARACTER VARYING, c10 TIME, c11 TIMESTAMP, c13 INTERVAL, c14 STRUCT<a INT>, c15 INT[], c16 CHARACTER VARYING[]);
+CREATE TABLE alltypes2 (c1 BOOLEAN, c2 SMALLINT, c3 INT, c4 BIGINT, c5 REAL, c6 DOUBLE, c7 NUMERIC, c8 DATE, c9 CHARACTER VARYING, c10 TIME, c11 TIMESTAMP, c13 INTERVAL, c14 STRUCT<a INT>, c15 INT[], c16 CHARACTER VARYING[]);
+CREATE MATERIALIZED VIEW m0 AS SELECT (INT '83') AS col_0, ((SMALLINT '330') & (SMALLINT '582')) AS col_1, (FLOAT '810') AS col_2 FROM alltypes2 AS t_0 GROUP BY t_0.c16, t_0.c14, t_0.c9;
+CREATE MATERIALIZED VIEW m1 AS SELECT t_0.c1 AS col_0, TIME '07:31:30' AS col_1, (TIME '06:31:30' - (INTERVAL '0')) AS col_2, true AS col_3 FROM alltypes1 AS t_0 GROUP BY t_0.c1, t_0.c7, t_0.c9, t_0.c6;
+CREATE MATERIALIZED VIEW m2 AS SELECT TIMESTAMP '2023-02-16 07:30:30' AS col_0, (0) AS col_1, TIMESTAMP '2023-02-16 07:30:30' AS col_2 FROM alltypes2 AS t_0 GROUP BY t_0.c13, t_0.c2, t_0.c1 HAVING ((-2599614897320801085) >= (coalesce(NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, (FLOAT '207'), NULL)));
+CREATE MATERIALIZED VIEW m3 AS WITH with_0 AS (SELECT (953) AS col_0, TIMESTAMP '2023-02-08 04:45:17' AS col_1, min(TIME '07:30:30') AS col_2, (572) AS col_3 FROM supplier AS t_1 WHERE true GROUP BY t_1.s_comment HAVING true) SELECT (SMALLINT '336') AS col_0, (SMALLINT '13') AS col_1, 'dsiNHXkoXU' AS col_2 FROM with_0 WHERE true;
+CREATE MATERIALIZED VIEW m4 AS WITH with_0 AS (SELECT (FLOAT '976') AS col_0, (CASE WHEN false THEN (268) ELSE (-2375603950700629112) END) AS col_1, TIME '07:30:30' AS col_2 FROM lineitem AS t_1 WHERE ((1) <> (FLOAT '92')) GROUP BY t_1.l_orderkey, t_1.l_linenumber, t_1.l_partkey, t_1.l_comment, t_1.l_shipmode, t_1.l_returnflag, t_1.l_quantity, t_1.l_tax HAVING false) SELECT TIMESTAMP '2023-02-13 11:07:51' AS col_0, '3Zgh7k8v4s' AS col_1, (SMALLINT '-32768') AS col_2, (REAL '815') AS col_3 FROM with_0;
+CREATE MATERIALIZED VIEW m5 AS SELECT (SMALLINT '32767') AS col_0 FROM auction AS t_0 GROUP BY t_0.category, t_0.description, t_0.seller, t_0.initial_bid, t_0.date_time;
+CREATE MATERIALIZED VIEW m7 AS SELECT (TIMESTAMP '2023-02-15 07:31:30' + (INTERVAL '256238')) AS col_0, (TIME '19:15:27' - (INTERVAL '-3600')) AS col_1 FROM nation AS t_0 WHERE false GROUP BY t_0.n_regionkey, t_0.n_name;
+CREATE MATERIALIZED VIEW m8 AS SELECT (FLOAT '110') AS col_0, (((SMALLINT '593') << (SMALLINT '0')) >> (INT '437')) AS col_1 FROM m2 AS t_0 WHERE true GROUP BY t_0.col_1;
+CREATE MATERIALIZED VIEW m9 AS WITH with_0 AS (SELECT (208) AS col_0, (253) AS col_1 FROM bid AS t_1 WHERE CAST((INT '617') AS BOOLEAN) GROUP BY t_1.extra, t_1.date_time, t_1.auction, t_1.price) SELECT TIMESTAMP '2023-02-16 07:31:30' AS col_0, 'pvwvrazizs' AS col_1, 'lgGTZkuYLo' AS col_2 FROM with_0 WHERE true;
```

Comment:
Start by manually checking in the frozen query first...
We can setup a weekly cron to upload and maintain the artifact in buildkite instead.

Reply:
Actually, we should fix that in this PR too, since we should generate 32 sets of queries and ddl, and it's not a good idea to check all these in.

Reply:
I guess let's ensure stability first, and then setup the weekly cron for this... for now just zip up 32 queries.